less/navigation.less

```diff
@@ -1 +1,51 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+.header{
+    padding:2% 0;
+    border-bottom:@border;
+    
+    @media @mobile{
+        padding:0;
+    }
+
+    h1{
+        margin: 3% 0 2% 0;
+    }
+
+    div{
+        margin: 0 auto;
+        width:800px;
+        .flexbox();
```

Comment:
Check out `mixins.less` for how I changed `.flexbox()`

I would change 
```
.flexbox();
justify-content: space-between;
```

to
```
.flexbox(space-between);
```

space-between will then be applied to justify-content, so you won't need to type the justify-content again.